BPStatusBarAlert/Classes/DeviceDimensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DeviceDimensions.swift
+//  
+//
+//  Created by Sameh Salama on 11/28/17.
+//
+
+import UIKit
+
+public class DeviceDimensions {
+    
+    fileprivate var iPhoneXDeviceHeight:CGFloat = 2436
```

Comment:
- iPhoneXDeviceHeight is constant value, so `let` is more suitable this case
- My preferred code convention is: `fileprivate let iPhoneXDeviceHeight: CGFloat = 2436`

Reply:
done